lib/csv/writer.rb

```diff
@@ -149,5 +151,10 @@ def quote(field)
         end
       end
     end
+
+    def apply_converters_to(row, lineno)
+      converter = @options[:writer_fields_converters]
```

Comment:
Could you avoid fetching from `@options` each time? We need high performance.
Instance variable is faster than `Hash#[]`.